src/main/java/bio/terra/cda/common/exception/ErrorReportException.java

```diff
@@ -20,7 +20,7 @@ public ErrorReportException(String message) {
 
   public ErrorReportException(String message, Throwable cause) {
     super(message, cause);
-    this.causes = null;
+    this.causes = List.of(cause.toString());
```

Comment:
This isn't quite correct as each throwable can itself have another cause, and we don't know in general which cause is the most interesting. One solution is to collect all causes by following up the chain to the root.

Reply:
+1 What I intended was to be able to return the BQ error with the error message too, but this does not do that. What would be a mechanism for that? The console prints out the BQ error though.

Reply:
I created a PR that fixes this, at least for this case.

Reply:
For my change to work properly this line should be reverted, since the fix depends on `causes` to be null.

Reply:
Is this why the message is repeated in causes?

Reply:
The way it is right now, the top level message is shown twice (if `causes` is not specified by the exception). `ErrorReport.message` shows it once and `ErrorReport.causes` will have it as the first cause too. Maybe this isn't what we want; it could be changed to not show the top level exception in `causes`.